content/docs/for-platform-operators/tutorials/lifecycle/_index.md

```diff
@@ -5,4 +5,177 @@ expand=true
 include_summaries=true
 +++
 
-Coming soon! In the meantime check out this excellent [blog post](https://medium.com/buildpacks/unpacking-cloud-native-buildpacks-ff51b5a767bf) from our community.
+A `platform` orchestrates builds by invoking the [lifecycle][lifecycle] binary together with buildpacks and application source code to produce a runnable `OCI image`.
+
+<!--more-->
+
+The majority of Buildpack users use platforms, such as [pack][pack] and [kpack][kpack], to run Buildpacks and create `OCI images`. However this might not be desireable especially for users maintaining their own platforms and seeking more control over how the underlying Buildpack `lifecycle phases` are executed.
+
+In this step-by-step tutorial, you will build a `Bash` application without using any `platform` tooling like `pack` or `kpack`. You will also leverage the individual `lifecycle phases` to produce a runnable application image.
+
+## Prerequisites
+
+You'll need to clone a local copy of the following to get started:
+
+* The `lifecycle`: `git clone https://github.com/buildpacks/lifecycle`
+* The official `Buildpack.io` samples repo: `git clone https://github.com/buildpacks/samples`
+
+As previously mentioned, the `lifecycle` orchestrates `Buildpacks` then assembles the resulting artifacts into an `OCI image`.  The `lifecycle` is composed of a series of distinct `phases` that need to be executed to have the final image built and exported.
+
+## Overview
+
+Now that you’re set up, let’s build our `Bash` application and dive deeper into the `lifecycle` and its phases.
+
+### Build the lifecycle
+
+As a starting step, you need to build the `lifecycle` in order to use its phases. This could be done by navigating to the `lifecycle` directory and executing one of the following commands, depending on your system architecture.
+
+* `make build` for `AMD64` architectures
+* `make build-darwin-arm64` for `ARM64` architectures
```

Comment:
We may need to point out that `darwin-arm64` is specifically for Mac machines.  Do we also need to point out that the entire process is most easily followed on a Linux machine?

Reply:
added